R/plan.sh

```diff
@@ -38,9 +38,9 @@ pkg_deps=(
   core/xz
   core/zlib
 )
-pkg_bin_dirs=(lib64/R/bin)
-pkg_include_dirs=(lib64/R/include)
-pkg_lib_dirs=(lib64/R/lib)
+pkg_bin_dirs=(bin)
```

Comment:
@echohack [your PR](https://github.com/habitat-sh/core-plans/pull/2709/files#diff-6466e655a7d054bcdee1e1f4021f6646R41) fixed this by changing `lib64/R/bin` to `lib/R/bin` ...but here I was simply changing it to the pkg's `bin` directory. 
Is there any wisdom around preferring one directory over the other?

Reply:
I was using `lib/R/bin` because when I was messing around with the package, there were many more binaries dropped in that directory.

I think the `bin` directory is just a symlink over to `lib/R/bin`.

I honestly don't know enough about R to comment on exactly the right directory to link.